packages/truffle/test/scenarios/migrations/dryrun.js

```diff
@@ -5,35 +5,27 @@ const assert = require("assert");
 const Server = require("../server");
 const Reporter = require("../reporter");
 const sandbox = require("../sandbox");
-const Web3 = require("web3");
 
-describe("migrate (dry-run)", function() {
+describe("migrate (dry-run)", function () {
   let config;
-  let web3;
   const project = path.join(__dirname, "../../sources/migrations/success");
   const logger = new MemoryLogger();
 
   before(done => Server.start(done));
   after(done => Server.stop(done));
 
-  before(async function() {
+  before(async function () {
     this.timeout(10000);
     config = await sandbox.create(project);
     config.network = "development";
     config.logger = logger;
     config.mocha = {
       reporter: new Reporter(logger)
     };
-
-    const provider = new Web3.providers.HttpProvider("http://localhost:8545", {
```

Comment:
So was all this stuff just not used, I'm guessing?

Reply:
yep there was a bunch of extra stuff